src/tcgwars/logic/impl/gen7/UnbrokenBonds.groovy

```diff
@@ -4314,23 +4314,31 @@ public enum UnbrokenBonds implements LogicCardInfo {
         return pokemonTool (this) {
           text "If this card is attached to 1 of your Pokémon, discard it at the end of your opponent's turn." +
             "The [M] Pokémon this card is attached to takes 70 less damage from your opponent's attacks (after applying Weakness and Resistance)."
-          def eff
+          def eff1, eff2
           onPlay {reason->
-            eff=delayed{
+            eff1 = delayed {
               before APPLY_ATTACK_DAMAGES,{
```

Comment:
Does this reduce the 70 hp after applying weakness and resistance?  Just curious

Reply:
Yeah, `before APPLY_ATTACK_DAMAGES` reduces after W/R and `before APPLY_ATTACK_DAMAGES` and `after PROCESS_ATTACK_EFFECTS` would reduce before it.

Reply:
Huh interesting.  Thanks for clarifying